packages/core/src/types/define.ts

```
@@ -0,0 +1,87 @@
+    Name,
+    Definition,
+    PackedEnum<Definition>
+  > {
+  __type__?: () => 'Enum'
+}
+
+export interface ObjectType<
+  Name extends string,
+  Fields extends Record<string, Field<any, any, any>>,
+// eslint-disable-next-line ts/no-empty-object-type
+> extends BaseObject<Name, Fields, {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)